roles/setup_pgd/tasks/bdr_init_physical.yml

```diff
@@ -0,0 +1,55 @@
+---
+- name: Check pg_wal state
+  ansible.builtin.stat:
+    path: "{{ pg_data }}/pg_wal"
+  become: true
+  register: pg_wal_state
+  when: not initdb_executed
+
+- name: Prepare PGD backup command using bdr_init_physical
+  ansible.builtin.set_fact:
+    bdr_init_physical: "{{ bdr_init_physical }} --remote-dsn='{{ lead_primary_dsn }}' --log-file={{ pg_log }}/bdr_init_physicail.log"
+  when: not initdb_executed
+
+- name: Create PGD node using bdr_init_physical
+  ansible.builtin.shell: >
+    PGAPPNAME={{ inventory_hostname }} {{ bdr_init_physical }}
+  args:
+    creates: "{{ pg_data }}/PG_VERSION"
+  become: true
+  become_user: "{{ pg_owner }}"
+  async: 180000
```

Comment:
We've been removing these async lines because they are causing issue.  Don't we want to keep removing them?